invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html

```diff
@@ -30,14 +30,15 @@
 {% macro show_creatibutors(creatibutors, show_affiliations=False) %}
   {% for creatibutor in creatibutors %}
     {{ creatibutor_icon(creatibutor) }}
-    <span
+    <a
       class="ui tooltip-popup text-muted"
       {% if show_affiliations and creatibutor.affiliations %}
         data-content="{{ creatibutor.affiliations|join('; ', attribute='1') }}"
       {% endif %}
+      href="/search?q=metadata.creators.person_or_org.name{{ creatibutor.person_or_org.name}}"
```

Comment:
```suggestion
      href="{{ url_for('invenio_search_ui.search', q='metadata.creators.person_or_org.name:" creatibutor.person_or_org.name") }}"
```

Can you check that the following would work:

- Deposit a record with both an person and organistaion.
- Make sure both of them have spaces in their name.
- Once deposited, you  should be able to click the name, and get a hit in the search result.